install/db/dbmigrate_copyright-event.php

```diff
@@ -90,7 +94,7 @@ function insertDataInToEventTables($dbManager)
       $startTime = microtime(true);
       $dbManager->begin();
       $statementName = __METHOD__."insert from function".$i;
```

Comment:
If we replace the `$i` with `$table`, we can reuse the prepared statement more efficiently.
```suggestion
      $statementName = __METHOD__."insert from function".$table;
```